src/components/Transfer.tsx

```diff
@@ -61,6 +61,14 @@ function Transfer() {
   const { api, isApiReady } = sourceChainDetails.apiConnection;
   const balance = useBalance(api, account?.address || '');
 
+  const transformInput = (value: string) => {
+    const [actualValue, message] = evalUnits(value || '0');
+    setHelperText(message);
+    return actualValue && actualValue * planck;
+  };
+
+  const [transferInput, actualInput, setActualInput] = useDebounceState('', null, transformInput);
```

Comment:
Taking into account what @wirednkod is working on #204 where the input will be moved to the reducer logic , some of the features here might not be needed any more. The idea is to not use the `transformInput` callback and assign the same value to both states so the one going to the reducer will be processed there.

Reply:
```suggestion
  const [currentInput, debouncedInput, setInput] = useDebounceState('', null, transformInput);
```
I'd rename the variables to emphasise it's debounced.

Reply:
sure.